### PR TITLE
feat: update to bee v0.6.0

### PR DESCRIPTION
**Pull request by vojtechsimetka**
- [x] update to released `bee-js` version once released
- [x] remove upload for now (until postage stamps are added)

### CODE REVIEW

File: src/pages/files/index.tsx [hunk 1]
```diff
@@ -90,60 +66,29 @@ export default function Files(): ReactElement {
...
-              {uploadError && <FormHelperText error>{uploadError.message}</FormHelperText>}
-            </div>
-          </div>
-        </div>
-      )}
+      <Paper className={classes.root}>
+        <InputBase
+          className={classes.input}
+          placeholder="Enter swarm reference e.g. 0773a91efd6547c754fc1d95fb1c62c7d1b47f959c2caa685dfec8736da95c1c"
+          inputProps={{ 'aria-label': 'retriefe file from swarm' }}
```
Comment (agazso) [inline src/pages/files/index.tsx:77]:
```suggestion
          inputProps={{ 'aria-label': 'retrieve file from swarm' }}
```

File: src/hooks/accounting.ts [hunk 2]
```diff
@@ -93,9 +91,10 @@ export const useAccounting = (): UseAccountingHook => {
     if (isLoadingUncashed || !settlements.settlements || uncashedAmounts || error) return
 
     setIsloadingUncashed(true)
-    const promises = settlements.settlements.settlements.map(({ peer }) =>
-      beeDebugApi.chequebook.getPeerLastCashout(peer),
-    )
+    const promises = settlements.settlements.settlements
```
Comment (agazso) [inline src/hooks/accounting.ts:94]: This looks interesting 🤔

**Review by agazso — APPROVED** [review]
👍

File: src/hooks/accounting.ts (hunk 2 above)
Comment (vojtechsimetka) [reply src/hooks/accounting.ts:94]: Yeah this should be extracted into a provider and we can drop at least one of these. Not a part of this PR, but there is an issue for it and I want to address it soon #40

**Pull request merged**